hack/make/test-integration-cli

```diff
@@ -9,23 +9,24 @@ bundle_test_integration_cli() {
 
 # subshell so that we can export PATH without breaking other things
 (
-	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
+	SCRIPTDIR=$(dirname "$BASH_SOURCE")
```

Comment:
Just thinking that would make sure our usage of what `SCRIPTDIR` means stays consistent to be referring to the `hack` directory.